components/CounterContinent/CounterContinent.js

```diff
@@ -55,16 +55,18 @@ export default function CounterContinent({ countries, countriesInfo }) {
 
   return (
     <>
-      {continentCounts.map((continent) => (
-        <StyledDiv withBorder key={continent.name}>
-          <h3>{continent.name}</h3>
-          <p>
-            <StyledCounterContinent>
-              {continent.visitedCount} / {continent.totalCount}{" "}
-            </StyledCounterContinent>
-          </p>
-        </StyledDiv>
-      ))}
+      {continentCounts
+        .sort((a, b) => a.name.localeCompare(b.name))
```

Comment:
![image](https://user-images.githubusercontent.com/109807794/231228835-6067ca87-d668-4185-8b54-71eb1851126a.png)

Reply:
ah dammit, forgot that again.. not is is fixed! Thanks @Satttoshi 💪🏼